CMakeLists.txt

```diff
@@ -118,6 +118,11 @@ include(GNUInstallDirs)
 include(autocmake_safeguards)
 include(custom_color_messages)
 
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set(CMAKE_INSTALL_PREFIX "/usr/local/psi4" CACHE PATH "Install path" FORCE)
```

Comment:
Wait, why change the default install from ```/usr/local``` to ```/usr/local/psi4```?  We install everything into a ```psi4``` subdirectory:  ```$prefix/include/psi4```, ```$prefix/share/psi4```, ```$prefix/lib/psi4``` but the binaries stay in ```$prefix/bin```.  Changing the default from ```/usr/local``` means users have to modify their path to add ```/usr/local/psi4/bin```.